src/main/java/game/user/User.java

```diff
@@ -351,9 +351,11 @@ public int incEXP(int inc, boolean onlyFull) {
                     }
                     Pointer<Boolean> reachMaxLev = new Pointer<>(false);
                     if (ExpAccessor.tryProcessLevelUp(character, basicStat, inc, reachMaxLev)) {
+						if (character.getCharacterStat().getJob() == 0) {
```

Comment:
Your code does the same thing as what's on the master branch but with more code. I tested before pushing to master and it works fine so I'm assuming you somehow messed up applying the changes on master when testing.